php56/provision.sh

```diff
@@ -72,72 +61,32 @@ network_check() {
   fi
 }
 
-not_installed() {
-  dpkg -s "$1" 2>&1 | grep -q 'Version:'
-  if [[ "$?" -eq 0 ]]; then
-    apt-cache policy "$1" | grep 'Installed: (none)'
-    return "$?"
-  else
-    return 0
-  fi
-}
-
-print_pkg_info() {
-  local pkg="$1"
-  local pkg_version="$2"
-  local space_count
-  local pack_space_count
-  local real_space
-
-  space_count="$(( 20 - ${#pkg} ))" #11
-  pack_space_count="$(( 30 - ${#pkg_version} ))"
-  real_space="$(( space_count + pack_space_count + ${#pkg_version} ))"
-  printf " * $pkg %${real_space}.${#pkg_version}s ${pkg_version}\n"
-}
-
-package_check() {
-  # Loop through each of our packages that should be installed on the system. If
-  # not yet installed, it should be added to the array of packages to install.
-  local pkg
-  local pkg_version
-
-  for pkg in "${apt_package_check_list[@]}"; do
-    if not_installed "${pkg}"; then
-      echo " *" "$pkg" [not installed]
-      apt_package_install_list+=($pkg)
-    else
-      pkg_version=$(dpkg -s "${pkg}" 2>&1 | grep 'Version:' | cut -d " " -f 2)
-      print_pkg_info "$pkg" "$pkg_version"
-    fi
-  done
-}
-
 package_install() {
-  package_check
 
-  if [[ ${#apt_package_install_list[@]} = 0 ]]; then
-    echo -e "No apt packages to install.\n"
-  else
-
-    # Apply the PHP signing key
-    apt-key adv --quiet --keyserver "hkp://keyserver.ubuntu.com:80" --recv-key E5267A6C 2>&1 | grep "gpg:"
```

Comment:
This command is disappeared, it will be executed in VVV instead?

Reply:
Right now doesn't seems that is added https://github.com/Varying-Vagrant-Vagrants/VVV/search?q=apt-key&unscoped_q=apt-key

Reply:
yes it is, we use Ondrej packages, which are added here:

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/b3d1d0c8ca280142373b4ffb46ef3ccde4c369b7/provision/provision.sh#L379